core/src/test/java/nl/nn/adapterframework/pipes/CompareStringPipeTest.java

```diff
@@ -14,24 +15,12 @@ public CompareStringPipe createPipe() {
 
     @Test(expected = ConfigurationException.class)
     public void emptySessionKeys() throws ConfigurationException {
+        PipeForward forw = new PipeForward("lessthan", "/Users/apollo11/Desktop/iaf2/core/src/test/resources/Pipes");
+        pipe.registerForward(forw);
         pipe.setSessionKey1("");
         pipe.setSessionKey2("");
         pipe.configure();
     }
 
-    @Test
```

Comment:
> Why did you removed these 2 tests?

Thank you for pointing this out, I actually don't have a clue and cannot recall removing them, as this test does not belong to me. I'm going to readd these and I will try to be more careful to avoid errors like this and double check my pull requests